### PR TITLE
Feature/neo features

### PR DESCRIPTION
**Pull request by rockacola**
Implement "NEO Features" section of the homepage to fit newly (and easier) design (per Design Files v0.09)

Expect visual:

image

### CODE REVIEW

File: NeoWeb/Views/Home/_NeoFeatures.cshtml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@model List<NewsViewModel>
+@using Microsoft.AspNetCore.Mvc.Localization
+@inject IViewLocalizer Localizer
+@inject IHtmlLocalizer<SharedResource> SharedLocalizer
+
+@{ string[,] blocks = new string[7, 4] {
+  {
+      "/images/home/OneBlockFinality3D.svg", "One Block Finality", "dBFT consensus mechanism guarantees fast and efficient finality in a single block.", "technology#dbft-2.0"
```
Comment (chenzhitong) [inline NeoWeb/Views/Home/_NeoFeatures.cshtml:8]: Need to update resource files

**Review by chenzhitong — CHANGES_REQUESTED** [review]
Need to update resource files

**Comment by rockacola** [comment]
Hi @chenzhitong , can you provide some guidance on how to correctly perform l18n with arrayed data object at https://github.com/neo-ngd/neo.org/blob/cd9fadbcb30a6f1c85127d91aa920f90572329b7/NeoWeb/Views/Home/_NeoFeatures.cshtml#L6-L29

I've tried to add Localizer before adding string into the array (like below), but didn't work out for me.

```cs
@{ string[,] blocks = new string[7, 4] {
  {
      "/images/home/OneBlockFinality3D.svg", Localizer["One Block Finality"].Value, Localizer["dBFT consensus mechanism guarantees fast and efficient finality in a single block."].Value, "technology#dbft-2.0"
  },
  ...
};
```

Do I need to explicit reference to the resource file somewhere?

**Comment by chenzhitong** [comment]
Add `Views.Home._NeoFeatures.zh.resx` to Resources folder. @rockacola

**Comment by rockacola** [comment]
@chenzhitong I've updated the resource file and its usage. It is now ready for another review. Thank you.

**Pull request merged**